test/Microsoft.ML.Tests/TrainerEstimators/MatrixFactorizationTests.cs

```diff
@@ -253,6 +254,31 @@ public void MatrixFactorizationInMemoryData()
                 Assert.True(pred.Score != 0);
         }
 
+        internal class MatrixElementZeroBased256By256
+        {
+            // Matrix column index starts from 0 and is at most _synthesizedMatrixColumnCount.
+            [KeyType(_matrixColumnCount)]
+            public uint MatrixColumnIndex;
+            // Matrix row index starts from 0 and is at most _synthesizedMatrixRowCount.
+            [KeyType(_matrixRowCount)]
+            public uint MatrixRowIndex;
+            // The value at the MatrixColumnIndex-th column and the MatrixRowIndex-th row in the considered matrix.
+            public float Value;
+        }
+
+        internal class MatrixElementZeroBasedForScore256By256
+        {
+            // Matrix column index starts from 0 and is at most _synthesizedMatrixColumnCount.
+            // Contieuous=true means that all values from 0 to _synthesizedMatrixColumnCount are allowed keys.
+            [KeyType(_matrixColumnCount)]
+            public uint MatrixColumnIndex;
+            // Matrix row index starts from 0 and is at most _synthesizedMatrixRowCount.
+            // Contieuous=true means that all values from 0 to _synthesizedMatrixRowCount are allowed keys.
```

Comment:
>Contieuous [](start = 15, length = 10)

Contiguous? Continuous?

Reply:
Those `Contieuous`s are removed because they are not a part of the latest key type.

---
In reply to: [273743333](https://github.com/dotnet/machinelearning/pull/3124#discussion_r273743333) [](ancestors = 273743333)